src-db/database/sourcedata/OBUIAPP_PARAMETER.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<data>
+<!--E5212D089F664F2E9CCDB677981DCF20--><OBUIAPP_PARAMETER>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <OBUIAPP_PARAMETER_ID><![CDATA[E5212D089F664F2E9CCDB677981DCF20]]></OBUIAPP_PARAMETER_ID>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <AD_CLIENT_ID><![CDATA[0]]></AD_CLIENT_ID>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <AD_ORG_ID><![CDATA[0]]></AD_ORG_ID>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <AD_MODULE_ID><![CDATA[2EC4FFAFFE984592BA9859A8C9E25BF0]]></AD_MODULE_ID>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <NAME><![CDATA[Version]]></NAME>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <SEQNO><![CDATA[10]]></SEQNO>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <AD_REFERENCE_ID><![CDATA[95E2A8B50A254B2AAE6774B8C2F28120]]></AD_REFERENCE_ID>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <AD_REFERENCE_VALUE_ID><![CDATA[EA9C1CF85A4E4425880C9B4B5469E591]]></AD_REFERENCE_VALUE_ID>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <COLUMNNAME><![CDATA[version]]></COLUMNNAME>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <ISCENTRALLYMAINTAINED><![CDATA[Y]]></ISCENTRALLYMAINTAINED>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <FIELDLENGTH><![CDATA[0]]></FIELDLENGTH>
+<!--E5212D089F664F2E9CCDB677981DCF20-->  <ISMANDATORY><![CDATA[N]]></ISMANDATORY>
```

Comment:
Is it OK for the `Change Version` process to have its `version` parameter set as optional? Shouldn't it be mandatory?